src/components/NumberInput/NumberInput.stories.tsx

```
@@ -0,0 +1,38 @@
+    value: 10,
+    unit: 'px',
+    onChange: fn(),
+  },
+} satisfies Meta<typeof NumberInput>
+
+export default meta
+type Story = StoryObj<typeof meta>
+
+export const Default: Story = {
+  render: function Render(args) {
+    const [{}, updateArgs] = useArgs()
```

Biome (2.1.2)

[error] Unexpected empty object pattern.



(lint/correctness/noEmptyPattern)